vcell-core/src/main/java/cbit/vcell/mapping/NetworkTransformer.java

```diff
@@ -331,24 +335,24 @@ private void transform(SimulationContext simContext, SimulationContext transform
 			}
 			FakeReactionRuleRateParameter fakeKineticParam = FakeReactionRuleRateParameter.fromString(s);
 			if(fakeKineticParam != null) {
-				System.out.println("found fakeKineticParam "+fakeKineticParam.fakeParameterName);
+				lg.info("found fakeKineticParam "+fakeKineticParam.fakeParameterName);
 				continue;	// we get rid of the fake parameters we use as keys
 			}
 			throw new RuntimeException("unexpected parameter "+p.getName()+" in internal BNG processing");
 //			Expression exp = new Expression(p.getValue());
 //			exp.bindExpression(model.getRbmModelContainer().getSymbolTable());
 //			model.getRbmModelContainer().addParameter(p.getName(), exp, model.getUnitSystem().getInstance_TBD());
 		}
-		endTime = System.currentTimeMillis();
-		elapsedTime = endTime - startTime;
-		msg = "Adding " + outputSpec.getBNGParams().length + " parameters to model, " + elapsedTime + " ms";
-		System.out.println(msg);
+//		endTime = System.currentTimeMillis();
+//		elapsedTime = endTime - startTime;
+//		msg = "Adding " + outputSpec.getBNGParams().length + " parameters to model, " + elapsedTime + " ms";
+//		lg.info(msg);
 		
 		// ---- Species ------------------------------------------------------------------------------------------------------------
 		mathMappingCallback.setMessage("generating network: adding species...");
 		mathMappingCallback.setProgressFraction(progressFractionQuota/4.0f);
-		startTime = System.currentTimeMillis();
-		System.out.println("\nSpecies :");
+//		startTime = System.currentTimeMillis();
+		lg.info("\nSpecies :");

```

Comment:
Minor comment:

1. you may want to remove separate "title" log entries like lg.info("\nSpecies :") and lg.info("\nReactions :") and instead make each subsequent log message more self-contained.  For example, add a "Reaction" prefix to reaction-related log messages if needed.
2. details of processing should use mostly lg.debug() rather than lg.info() to not clog up the logs.  This is especially important during CI/CD testing when such logs make it hard to see the important stuff.